contrib/jwrapper/src/code_generator.nit

```diff
@@ -355,7 +355,7 @@ extern class {{{nit_type}}} in "Java" `{ {{{java_type.extern_equivalent}}} `}
 		end
 
 		file_out.write """
-	# Java constructor: {{{java_class}}}
+{{{t}}}# Java constructor: {{{java_class.class_type.extern_equivalent}}}
```

Comment:
There's no `t` here since constructors are always within a class, and thus always indented.